src/features/Dev/DevSlice.ts

```diff
@@ -0,0 +1,54 @@
+import { SagaIterator } from '@redux-saga/types';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { takeEvery, getContext, call, put } from 'redux-saga/effects';
+
+import { ConsecutiveBreachAction } from '../Breach';
+import { Sensor } from '../../common/services/Database/entities';
+import { DEPENDENCY, REDUCER } from '../../common/constants';
+
+interface CreatePayload {
+    sensor: Sensor;
+  }  
+
+const initialState = {};
+
+const reducers = {
+  generateBreachLogs: {
```

Comment:
Not much going on here ATM, but ideally will move logic to generate sensors etc. here also (basically a slice for managing state related to the `DevSettingsScreen`).